src/editpage/components/response/MessagesWindow.vue

```diff
@@ -180,6 +180,14 @@ export default {
           error: this.addError.message,
           file: this.addError.file,
         });
+        this.showTimeout = setTimeout(() => {
+          requestAnimationFrame(() => {
```

Comment:
And you might want to check for `newValue === true` (i.e. that `showingAddError` is `true`)
since now this might trigger and error when an error disappears